tests/browser/heda_settings_accounts_and_contacts_edit_mode.robot

```diff
@@ -0,0 +1,92 @@
+    Click Element                   //div[@class='slds-tabs--default']//span[contains(text(),'Academic Program')]
+
+    Wait Until Element Is visible   //div[@class='slds-tabs--default']//span[contains(text(),'Administrative')]
+    Click Element                   //div[@class='slds-tabs--default']//span[contains(text(),'Administrative')]
+
+    Wait Until Element Is visible   //div[@class='slds-tabs--default']//span[contains(text(),'Business Organization')]
+    Click Element                   //div[@class='slds-tabs--default']//span[contains(text(),'Business Organization')]
+
+    Wait Until Element Is visible   //div[@class='slds-tabs--default']//span[contains(text(),'Educational Institution')]
+    Click Element                   //div[@class='slds-tabs--default']//span[contains(text(),'Educational Institution')]
+
+    Wait Until Element Is visible   //div[@class='slds-tabs--default']//span[contains(text(),'Household Account')]
+    Click Element                   //div[@class='slds-tabs--default']//span[contains(text(),'Household Account')]
+
+    Wait Until Element Is visible   //div[@class='slds-tabs--default']//span[contains(text(),'Sports Organization')]
+    Click Element                   //div[@class='slds-tabs--default']//span[contains(text(),'Sports Organization')]
+
+    Wait Until Element Is visible   //div[@class='slds-tabs--default']//span[contains(text(),'University Department')]
+    Click Element                   //div[@class='slds-tabs--default']//span[contains(text(),'University Department')]
+
+    Capture Page Screenshot
+
+    Sleep                           10
+
+
+    Click Element                   //div[@class='slds-form-element__control']//span[contains(text(),'Academic Program')]
+    Click Element                   //div[@class='slds-form-element__control']//span[contains(text(),'Administrative')]
+    Click Element                   //div[@class='slds-form-element__control']//span[contains(text(),'Business Organization')]
+    Click Element                   //div[@class='slds-form-element__control']//span[contains(text(),'Educational Institution')]
+    Click Element                   //div[@class='slds-form-element__control']//span[contains(text(),'Household Account')]
+    Click Element                   //div[@class='slds-form-element__control']//span[contains(text(),'Household Account')]
+    Click Element                   //div[@class='slds-form-element__control']//span[contains(text(),'Sports Organization')]
+    Click Element                   //div[@class='slds-form-element__control']//span[contains(text(),'University Department')]
+    Capture Page Screenshot
+    Sleep                           5
+
+
+    #Save the changes
+    Select Checkbox                 //label[@class='slds-checkbox']//input[@type='checkbox']
+    Capture Page Screenshot
+    Click Element                   //div[@class='slds-button-group']//span[contains(text(), 'Save')]
+
+
+*** Keywords ***
+
+
+Get Random Contact Info
```

Comment:
This test suite specific keyword doesn't appear to be used in this test and should be removed.

Reply:
Removed unused keyword contact info